### DISCUSSION TITLE
Lots of "Reserved for future use" crates

### DISCUSSION THREAD

**Discussion by ossi-saukko** [rust-lang/crates.io#5597] [Q&A] [answered]
I accidentally bumped into many "reserved for future use" crates for a particular user in the crates.io service. Is this okay, or is this abuse of the system?

https://crates.io/users/r3v2d0g

**Comment by carols10cents** [comment, marked as answer]
This is not against our current policies. Any change to our policies would need to go through the RFC process; please see https://github.com/rust-lang/rfcs/pull/2614 for one such RFC that has somewhat stalled because there isn't clear consensus yet.

**Reply by Turbo87** [reply]
FWIW with https://github.com/rust-lang/rfcs/pull/3463 our policies have changes and we have removed quite a few name-squatting crates since then :)
